vision_agent_tools/models/florencev2.py

```diff
@@ -87,36 +88,56 @@ def __call__(
         self,
         task: PromptTask,
         image: Optional[Image.Image] = None,
+        images: Optional[List[Image.Image]] = None,
         video: Optional[VideoNumpy] = None,
         prompt: Optional[str] = "",
     ) -> Any:
```

Comment:
Can we make the return type clear?

Reply:
I reviewed the reason earlier they added `Any`, it is because `Florencev2`  has multiple tasks, and could produce multiple types of output, it is a bit difficult to map them all, I'll follow up with another pull request.